webapp/packages/core-connections/src/DatabaseConnection.ts

```diff
@@ -6,20 +6,21 @@
  * you may not use this file except in compliance with the License.
  */
 import { AUTH_PROVIDER_LOCAL_ID } from '@cloudbeaver/core-authentication';
-import type { DatabaseConnectionFragment } from '@cloudbeaver/core-sdk';
+import type { DatabaseConnectionFragment, DatabaseConnectionOriginFragment } from '@cloudbeaver/core-sdk';
 
 export type DatabaseConnection = DatabaseConnectionFragment;
+export type DatabaseConnectionOrigin = DatabaseConnectionOriginFragment;
 
-export function isLocalConnection(connection: DatabaseConnection): boolean {
-  if (!connection.origin) {
+export function isLocalConnection(connectionOrigin: DatabaseConnectionOrigin | DatabaseConnection): boolean {
+  if (!('origin' in connectionOrigin) || !connectionOrigin.origin) {
     return true;
   }
```

Comment:
this will produce wrong results:
pass connection object === always true even actually connection is not local

please transform this function to accept only origin object (ObjectOriginInfoFragment)

Reply:
done